<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mesero</class>
 <widget class="QWidget" name="Mesero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesero</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/imagenes/img/Logo.png</normaloff>:/imagenes/img/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="menuH" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuH{
	background-color: #4F7302;
	border: none;
}</string>
   </property>
   <widget class="QLabel" name="relojD">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;HelvLight&quot;;
color: #F29F05;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="menuL" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuL{
	background-color: #4F7302;
}</string>
   </property>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>-7</x>
      <y>520</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSalir{
	background: none;
	border: none;
	image: url(:/imagenes/img/logOff.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOrden">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>60</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnOrden{
	background: none;
	border: none;
	image: url(:/imagenes/img/tomarPedido.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>831</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QWidget" name="zona1">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>821</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="zona2">
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>511</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#toolBox{
	background-color: white;
	font: 10pt &quot;HelvLight&quot;;
}

#toolBox::tab{
	border-radius: 10px;
	background-color: #4F7302;
	font: 12pt &quot;HelvLight&quot;;
	color: white;
}

#toolBox::tab:Hover{
	background-color: #A9BF04;
}

#page{
	background-color: white;
	font: 10pt &quot;HelvLight&quot;;
}

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>410</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Entradas</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>471</width>
         <height>391</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>410</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Plato fuerte</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_3">
      <attribute name="label">
       <string>Postre</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_4">
      <attribute name="label">
       <string>Bebidas</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="zona3"/>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
